### ISSUE TITLE
Tooling to create json files

### ISSUE THREAD

**Issue by bjartek** [onflow/flow-interaction-template-service#8]
Is there tools or planned tools that ensbles a dev to point to a directory and generate these json files?

**Comment by srinjoyc** [comment]
Yes! We will be releasing them shortly (CLI + web interfaces)! In the meantime, @prpatel05  made a quick primitive way to generate them here https://flow-runner-prpatel05.vercel.app/ (https://github.com/prpatel05/flow-runner)
